modules/features/manatime_feature_tasks/manatime_feature_tasks.install

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @file
+ * Adding path aliases.
```

Comment:
I'd change this comment to something more generic (this file could have more than this function).
Something like: "Install functions for tasks feature" or be creative.